pytorch/inference/docker/1.10/py3/sdk1.17.1/Dockerfile.neuron

```diff
@@ -107,7 +107,7 @@ RUN pip install --no-cache-dir -U \
 
 RUN pip install neuron-cc[tensorflow] --extra-index-url https://pip.repos.neuron.amazonaws.com \
  && pip install "torch-neuron>=1.10.1,<1.10.2" --extra-index-url https://pip.repos.neuron.amazonaws.com \
- && pip install torchserve==$TS_VERSION \
+ && pip install torchserve-nightly==2022.3.23.post2 \
```

Comment:
This is great! Please also comment out the ARG for TS_VERSION on this dockerfile to prevent confusion at a later date.